includes/admin/promos/notices/class-license-upgrade-notice.php

```diff
@@ -175,13 +199,22 @@ protected function _display() {
 
 			} elseif ( Pass_Manager::pass_compare( $this->pass_manager->highest_pass_id, Pass_Manager::EXTENDED_PASS_ID, '>=' ) ) {
 
+				if ( ! $this->has_affiliate_wp_license() ) {
 				// Extended pass or higher.
-				printf(
-				/* Translators: %1$s opening anchor tag; %2$s closing anchor tag */
-					__( 'Grow your business and make more money with affiliate marketing. %1$sGet AffiliateWP%2$s', 'easy-digital-downloads' ),
-					'<a href="' . esc_url( add_query_arg( $this->query_args( 'extended-pass', $source ), 'https://affiliatewp.com/?ref=743' ) ) . '" target="_blank">',
-					'</a>'
-				);
+					printf(
+					/* Translators: %1$s opening anchor tag; %2$s closing anchor tag */
+						__( 'Grow your business and make more money with affiliate marketing. %1$sGet AffiliateWP%2$s', 'easy-digital-downloads' ),
+						'<a href="' . esc_url( add_query_arg( $this->query_args( 'extended-pass', $source ), 'https://affiliatewp.com/?ref=743' ) ) . '" target="_blank">',
+						'</a>'
+					);
+				} elseif( ! $this->has_mi_license() ) {
+					printf(
+					/* Translators: %1$s opening anchor tag; %2$s closing anchor tag */
+						__( 'Gain access to powerful insights to grow your traffic and revenue. %1$sGet MonsterInsights%2$s', 'easy-digital-downloads' ),
+						'<a href="' . esc_url( 'https://www.monsterinsights.com?utm_campaign=xsell&utm_source=eddplugin&utm_content=top-promo' ) . '" target="_blank">',
```

Comment:
```suggestion
						'<a href="' . esc_url( 'https://monsterinsights.com?utm_campaign=xsell&utm_source=eddplugin&utm_content=top-promo' ) . '" target="_blank">',
```
Minor, but following the link, there is no www.

Reply:
Thanks, that's the link they gave me :P